scripts/coverage.sh

```diff
@@ -1,37 +1,29 @@
 #!/bin/sh
-# Runs tests and collects code coverage to ./cov.info file
+# Runs tests and collects code coverage to ./lcov.info file
 
 set -e # aborts as soon as anything returns non-zero exit status
 
 
 PROJECT_DIR=$(pwd -P)
 
-COVERAGE_FILE="$PROJECT_DIR"/cov.info
-INSTRUMENTED_CODE_DIR="$PROJECT_DIR"/src-cov
+COVERRAGE_DIR="$PROJECT_DIR"/coverage
+COVERAGE_FILE="$PROJECT_DIR"/coverage/lcov.info
```

Comment:
This change breaks `npm run coveralls`

Reply:
Fixed in https://github.com/apiaryio/dredd-transactions/pull/126/commits/9af69927799d273273e7f7be7ee08dde2a649c7f.